common/src/main/java/com/wynntils/features/trademarket/TradeMarketBulkSellFeature.java

```diff
@@ -56,25 +56,28 @@ public void onSellDialogueUpdated(ContainerSetSlotEvent.Pre e) {
         if (!(McUtils.mc().screen instanceof ContainerScreen containerScreen)) return;
 
         StyledText title = StyledText.fromComponent(containerScreen.getTitle());
-        if (!title.equalsString(SELL_DIALOGUE_TITLE, PartStyle.StyleType.NONE)) return;
+        // The sell title changes based on the state of the screen but always contains this string
+        if (!title.contains(SELL_DIALOGUE_TITLE)) return;
 
         StyledText amountItemName = StyledText.fromComponent(
                 containerScreen.getMenu().getSlot(AMOUNT_ITEM_SLOT).getItem().getHoverName());
         if (!amountItemName.equalsString(CLICK_TO_SET_AMOUNT, PartStyle.StyleType.NONE)) return;
 
         String soldItemName = getSoldItemName(containerScreen);
+        System.out.println(soldItemName);
 
         removeSellButtons(containerScreen);
 
         if (soldItemName == null) return;
 
         addSellButtons(containerScreen, soldItemName);
+        System.out.println("Added sell buttons");
     }
 
     @SubscribeEvent
     public void onChatMessage(ChatMessageReceivedEvent e) {
         if (!sendAmountMessage) return;
-        if (!e.getOriginalStyledText().equalsString(TYPE_SELL_AMOUNT, PartStyle.StyleType.NONE)) return;
+        if (!e.getOriginalStyledText().contains(TYPE_SELL_AMOUNT)) return;
```

Comment:
StartsWIth perhaps? although I wouldn't be sure the symbols are always the long one, maybe it's a "second" message in a line, so perhaps adapt the regex?

Reply:
For this, they send a block of 4 lines, and the part I'm using .contains on is guaranteed to be the second line
![image](https://github.com/user-attachments/assets/64ab858c-9ff4-44ea-984b-a95954852310)

So what's there now is actually the shorter symbols, and it's guaranteed to be the shorter symbols. That's also why I didn't use startswith

Reply:
Okay, sure.